doc/release-notes/11865-restored-suppress-dataverse-field

```diff
@@ -0,0 +1,9 @@
+### Restored suppression of the host dataverse field
```

Comment:
Since the release note is what users see, I'd suggest changing the title to "Suppression of the Host Dataverse field" and removing the second (for developers) and third (no longer true) paragraphs.